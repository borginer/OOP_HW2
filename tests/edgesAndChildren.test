CreateGraph G
CreateNode A 1
CreateNode B 2
CreateNode C 3
AddNode G A
AddNode G B
AddNode G C
AddEdge G A B
AddEdge G A C
ListChildren G A
