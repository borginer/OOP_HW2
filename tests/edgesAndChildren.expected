created graph G
created node A with cost 1
created node B with cost 2
created node C with cost 3
added node A to G
added node B to G
added node C to G
added edge from A to B in G
added edge from A to C in G
the children of A in G are: B C
